dss_datamover/utils/utility.py

```diff
@@ -181,7 +181,7 @@ def remoteExecution(host, username, password="", cmd="", blocking=False):
         return client, stdin, stdout, stderr
 
 
-def get_s3_prefix(logger, nfs_cluster, prefix=None):
+def get_s3_prefix(logger, nfs_cluster, prefix):
```

Comment:
what is the need for this change?

Reply:
this was for the both the SonarCloud complaint as well as a potential bug. There is an underlying function that uses the prefix that is passed in here. Therefore, a prefix must be passed in and cannot be none

Reply:
I initially thought this was the issue SonarCloud was complaining about 2v3 args, I will change it back